plugins/microsoft_atp/plugin.spec.yaml

```
@@ -1468,6 +1469,29 @@ actions:
         type: machine_action
         required: true
         example: {"collect_investigation_package_response":  {"requestorComment": "Investigation package collected via InsightConnect" ,"status": "Pending" ,"type": "CollectInvestigationPackage" ,"commands": [] ,"creationDateTimeUtc": "2021-02-25T13:53:29.1889041Z" ,"requestSource": "PublicApi" ,"id": "7de39b39-107e-4556-855b-25cf652835ef" ,"lastUpdateDateTimeUtc": "2021-02-25T13:53:29.1889041Z" ,"machineId": "8de370ca0e0e58ff2c2513bbc16f632ffa6e6024" ,"requestor": "b6e46392-61b9-48c0-ada3-63e3cd30d95b" ,"@odata.context": "https://api.securitycenter.windows.com/api/$metada..." ,"computerDnsName": "msedgewin10" ,"errorHResult":  0 }}
+  update_alert:
+    title: Update Alert
+    description: Updates properties of existing Alert
+    input:
+      alert_id:
+        title: Alert ID
+        description: ID of Alert to Update
```

Generally try and keep descriptions from being capitalised as if they were titles
```suggestion
        description: ID of alert to update
```